Fee Market Meetings/Meeting 08.md

```diff
@@ -0,0 +1,293 @@
+Stability. 
+
+**JosephC**: What is the accuracy? What are some heuristics that you are thinking of?
+
+**Ansgar Dietrichs**: well again something basically where it is like a very simple way of
+calculating some expected  future and effective minor mining tips. So for example you take the current base fee and then you take for example historically you always maybe the mempool  always keeps track of the variability  within the last I do not know 24 hours or something or it
+could be a different approach but then you just do it very simple. For example you just do
+like 50% of the current basically plus, like 25% of one sigma above and one sigma below basically just to give you some idea. Right again that might be as simple as that sounds like that might already be good enough that you have liked. It is not just the count base fee but you also
+like take into account like the higher one and the lower one or something and then you just update that maybe even just once every. So many blocks and as long as you can do that efficiently and as long as basically that is good enough that like on the high end your sorting is
+never is still absolutely like perfectly precise. But again I am right now not confident enough that this would work already. But that basically would be like currently, like one of my candidates for like a very simple heuristic that might be efficient but still good enough, yeah if that makes sense thanks. 
+
+**JosephC**: Yeah that does make sense.
+
+**Tim Beiko**: Yeah, cool anything else on the transaction pool management. 
+
+## 1.2 Update on the large state testnet
+
+**Tim Beiko**: Okay next up Abdel do you want to give a quick update on the large state testnet generator and where we are at?
+
+**Abdulhamid Bakhta**: Okay, yes so we currently have set up the new testnet so, this will be a
```

Comment:
```suggestion
**Abdelhamid Bakhta**: Okay, yes so we currently have set up the new testnet so, this will be a
```

Can you please update throughout? Thanks!

Reply:
Thanks for review . 
I have updated the notes with your suggestions.